test/e2e/fixtures/kube/core.k8s.io_services.yaml

```diff
@@ -54,9 +54,11 @@ spec:
                   description: healthCheckNodePort specifies the healthcheck nodePort for the service. If not specified, HealthCheckNodePort is created by the service api backend with the allocated nodePort. Will use user-specified nodePort value if specified by the client. Only effects when Type is set to LoadBalancer and ExternalTrafficPolicy is set to Local.
                   format: int32
                   type: integer
-                ipFamily:
-                  description: ipFamily specifies whether this Service has a preference for a particular IP family (e.g. IPv4 vs. IPv6).  If a specific IP family is requested, the clusterIP field will be allocated from that family, if it is available in the cluster.  If no IP family is requested, the cluster's primary IP family will be used. Other IP fields (loadBalancerIP, loadBalancerSourceRanges, externalIPs) and controllers which allocate external load-balancers should use the same IP family.  Endpoints for this Service will be of this family.  This field is immutable after creation. Assigning a ServiceIPFamily not available in the cluster (e.g. IPv6 in IPv4 only cluster) is an error condition and will fail during clusterIP assignment.
-                  type: string
+                ipFamilies:
```

Comment:
Do we really need this change ? Is it related to the fact that you've been using a higher release of Kube when CRD-pulling this file ?

Reply:
it is to fix the failure in e2e-shared in ci. In e2e-shared in ci. the service schema in kind uses ipFamilies, which is inconsistent with what we run e2e using another workspace for synctarget.